### DISCUSSION TITLE
What are Supabase's authentication headers?

### DISCUSSION THREAD

**Discussion by WilliamScott-AdventGX** [supabase/supabase#19317] [Questions] [answered]
I plan to migrate an existing application to use GraphQL. I am successfully querying the database now but **need to send user authentication data too (for RLS rules to work).** 

I've read the [GraphQL documentation for Supabase](https://supabase.com/docs/guides/graphql) but nothing on how to send user data.

I imagine it's notated somewhere else in Supabase's docs but no luck finding it. Any help would be great! 

Extra information:
- Using Supabase-JS
- Using Apollo-Angular (a wrapper for Apollo)

**Comment by GaryAustin1** [comment, marked as answer]
This covers details https://supabase.com/docs/learn/auth-deep-dive/auth-deep-dive-jwts .
Not specific to graphql but shows the headers apikey with anon or service_role jwt and Authorization with 'Bearer userjwt'.
The user jwt is the access_token in the session.

**Reply by WilliamScott-AdventGX** [reply]
Not tested but certainly the answer. Looks like `Authorization` with `Bearer ${jwt}` will be the solution. Much appreciated!
